Makefile

```diff
@@ -32,6 +34,10 @@ binary: build/Binary
 		@echo "[BINARY]"
 		@$(MAKE) -C build/Binary
 
+install: build/Release
+		@echo "[INSTALL] Release"
+		@$(MAKE) -C build/Release install
+
```

Comment:
Please add install to the help-section as well.

Reply:
Thx for fixing this. However one should avoid force pushing, if not strictly necessary. This should only be done if false changes have to be overridden in order to proceed. In this case an additional patch-commit would have done the job as well.